resources/user.rb

```diff
@@ -21,6 +21,7 @@
 
 property :host,           String,   default: 'localhost'
 property :port,           Integer,  default: 3000
+property :url_path_prefix, [String, nil], default: nil
```

Comment:
``Properties have a nil value by default so there is no need to set the default value to nil.`` This needs to be addressed as well.